### PR TITLE
Add more detailed logger setup

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: conda_forge_tick/auto_tick.xsh [hunk 1]
```diff
@@ -175,16 +180,16 @@ def main(args=None):
...
                         ts = c['reset']
-                        print('API timeout, API returns at')
-                        print(datetime.datetime.utcfromtimestamp(ts)
-                              .strftime('%Y-%m-%dT%H:%M:%SZ'))
+                        logger.info('API timeout, API returns at')
+                        logger.info(datetime.datetime.utcfromtimestamp(ts)
+                                    .strftime('%Y-%m-%dT%H:%M:%SZ'))
                         break
             except Exception as e:
                 print('NON GITHUB ERROR')
```
Comment (CJ-Wright) [inline conda_forge_tick/auto_tick.xsh]: Could this get logger-ified?

Comment (mariusvniekerk) [reply conda_forge_tick/auto_tick.xsh]: ah, missed one

File: conda_forge_tick/auto_tick.xsh [hunk 2]
```diff
@@ -81,15 +85,15 @@ def run(attrs, migrator, feedstock=None, protocol='ssh',
...
+        logger.critical("Failed to migrate %s, %s", $PROJECT, attrs.get('bad'))
         rm -rf @(feedstock_dir)
         return False, False
 
     # rerender, maybe
     with indir(feedstock_dir), ${...}.swap(RAISE_SUBPROC_ERROR=False):
         git commit -am @(migrator.commit_message())
         if rerender:
-            print('Rerendering the feedstock')
+            logging.info('Rerendering the feedstock')
```
Comment (justcalamari) [inline conda_forge_tick/auto_tick.xsh]: Should this be `logger.info`?

**Comment by mariusvniekerk** [comment]
Sorry seems i accidentally pushed a branch to here

**Comment by CJ-Wright** [comment]
Nbd it's not a feedstock! :) Are you ready for merge?

**Comment by scopatz** [comment]
Thanks @mariusvniekerk!

**Pull request merged**